packages/databricks-vscode/src/test/e2e/utils/commonUtils.ts

```diff
@@ -200,21 +200,27 @@ export async function waitForInput() {
 }
 
 export async function waitForLogin(profileName: string) {
+    console.log("Waiting for login");
```

Comment:
do you want to keep the `console.log` here?

Reply:
It's e2e test utility - the more logs there, the easier it is to debug them